src/ui/views/session/discover/DiscoverView.tsx

```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import { Room } from "@thirdroom/hydrogen-view-sdk";
+
+import { Content } from "../../../atoms/content/Content";
+import { Header } from "../../../atoms/header/Header";
+import { HeaderTitle } from "../../../atoms/header/HeaderTitle";
+import ExploreIC from "../../../../../res/ic/explore.svg";
+import ChevronRightIC from "../../../../../res/ic/chevron-right.svg";
+import { Icon } from "../../../atoms/icon/Icon";
+import { Window } from "../../components/window/Window";
+import { DiscoverHome } from "./DiscoverHome";
+import { SegmentControl } from "../../../atoms/segment-control/SegmentControl";
+import { SegmentControlItem } from "../../../atoms/segment-control/SegmentControlItem";
+import { usePowerLevels } from "../../../hooks/usePowerLevels";
+import { useHydrogen } from "../../../hooks/useHydrogen";
+import { DiscoverAdmin } from "./DiscoverAdmin";
+import { DiscoverAll } from "./DiscoverAll";
+import { DiscoverCreator } from "./DiscoverCreator";
+import "./DiscoverView.css";
+
+export enum RepositoryEvents {
+  FeaturedWorlds = "tr.repository_room.featured_worlds",
```

Comment:
We should determine the correct event prefix for this. If it's a vendor event then it should be `io.thirdroom.repository_room.featured_worlds`. Otherwise it should have an MSC created and use an MSC prefix.

Reply:
@turt2live has suggested we open an MSC and use the `org.matrix.` prefix for this. @ajbura can you open an MSC with the current event details and replace the prefixes in this PR.